SCClassLibrary/JITLib/Patterns/Pdef.sc

```diff
@@ -416,10 +419,13 @@ Tdef : TaskProxy {
 	storeArgs { ^[key] }
 
 	copy { |toKey|
-		if(key == toKey) { Error("cannot copy to identical key").throw };
+		if(toKey.isNil or: { key == toKey }) { Error("can only copy to new key (key is %)".format(toKey)).throw };
 		^this.class.new(toKey).copyState(this)
 	}
 
+	dup { |n = 2| ^{ this }.dup(n) } // avoid copy in Object::dup
+
```

Comment:
Is there a reason for the extra newline here?

Reply:
it reads much better to me. I only omit the newlines for long lists of one-liners.

Reply:
sorry, I understand now.